src/ui/components/ResultsTable.jsx

```diff
@@ -32,59 +27,8 @@ class ResultsTable extends Component {
     displayIsoforms: false,
   };
 
-  componentDidMount() {
-    this.loadResults();
-  }
-
-  componentDidUpdate() {
-    if (this.state.results === null) {
-      this.loadResults();
-    }
-  }
-
-  loadResults = () => {
-    const { params, history, clearSearchTerm } = this.props;
-    const apiURI = `${API_URL}/mappings`;
-
-    axios
-      .get(apiURI, { params })
-      .then(({ data, status }) => {
-        if (status === 204) {
-          clearSearchTerm(() => {
-            history.push(`${BASE_URL}/no-results`);
-          });
-
-          return;
-        }
-
-        const onlyIsoforms = data.results.every(d =>
-          d.entryMappings.every(mapping => !mapping.uniprotEntry.isCanonical));
-
-        const groupedResults = this.groupByIsoform(data.results);
-
-        this.setState({
-          params: this.props.params,
-          facets: data.facets,
-          results: groupedResults,
-          totalCount: data.count,
-          displayIsoforms: onlyIsoforms,
-        });
-      })
-      .catch((e) => {
-        console.log(e);
-        history.push(`${BASE_URL}/error`);
-      });
-  };
-
-  groupByIsoform = results =>
-    results.map(row => ({
-      taxonomy: row.taxonomy,
-      canonical: row.entryMappings.filter(mapping => mapping.uniprotEntry.isCanonical === true),
-      isoforms: row.entryMappings.filter(mapping => mapping.uniprotEntry.isCanonical === false),
-    }));
-
   toggleShowIsoforms = () => {
-    this.setState({ displayIsoforms: !this.state.displayIsoforms });
+    this.setState({ displayIsoforms: !this.props.displayIsoforms });
```

Comment:
The isoform show button works, but not the hide button anymore. Probably something to do with the state?